Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.5
+
+COPY . /app
+WORKDIR /app
+RUN pip install -e .
```

Comment:
Why `-e`? I think if we're going to make a production ready solution, it'd be better to make an actual installation. What do you think?